pico-imx7d: Enable loading kernel from rootfs partition

vanmaegima opened the pull request:
Replace fatload support for kernel loading with the fs generic loading interface
and enable booting from a single partition to support booting with a wic image.

These are needed since the wic scripts available do not generate a Boot (FAT)
partition.

Signed-off-by: Vanessa Maegima <vanessa.maegima@nxp.com>

otavio commented on include/configs/pico-imx7d.h:
```diff
@@ -38,20 +38,20 @@
-	"loadimage=fatload mmc ${mmcdev}:${mmcpart} ${loadaddr} ${image}\0" \
-	"loadfdt=fatload mmc ${mmcdev}:${mmcpart} ${fdt_addr} ${fdt_file}\0" \
+	"loadimage=ext4load mmc ${mmcdev}:${mmcpart} ${loadaddr} ${image}\0" \
+	"loadfdt=ext4load mmc ${mmcdev}:${mmcpart} ${fdt_addr} ${fdt_file}\0" \
```
This can be reworked and send upstream. Use 'load mmc' so it uses the fs generic interface and is agnostic.

After this commit, make another commit that adapts the code to the single partition use.

vanmaegima committed:
pico-imx7d: Replace fatload command

Replace fatload with the fs generic loading interface ('load' command).

Signed-off-by: Vanessa Maegima <vanessa.maegima@nxp.com>

vanmaegima committed:
pico-imx7d: Enable booting from single partition

Update 'fdt_file' and 'image' location and change rootfs mmc partition
in 'finduuid' to support booting from a single partition.

Signed-off-by: Vanessa Maegima <vanessa.maegima@nxp.com>

vanmaegima commented:
Pull request updated.
